habitat/plan.ps1

```diff
@@ -1,3 +1,8 @@
+
+# debugging - jfm
+$ErrorActionPreference = 'Continue'
```

Comment:
Could we remove this debugging change now?

Reply:
Done